### PR TITLE
Add api for projects

### PR DESCRIPTION
**Pull request by ondrejbartas**

### CODE REVIEW

**Comment by tomaass** [comment]
TODO
- [x] Add Active Model Serializer that adds locale to project from its relationship

File: app/controllers/api_frontend/v1/projects_controller.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
+module APIFrontend
+  module V1
+    class ProjectsController < ApiController
+      before_action :set_project, only: [:show, :update, :destroy]
+
+      # GET /projects
+      def index
+        @projects = policy_scope(Project)
+        respond_to do |format|
```
Comment (ondrejbartas) [inline app/controllers/api_frontend/v1/projects_controller.rb]: what about using responders and `respond_with @projects`??

Comment (ondrejbartas) [reply app/controllers/api_frontend/v1/projects_controller.rb]: check api in cheddar

**Comment by ondrejbartas** [comment]
othersise 👍

File: app/controllers/api_frontend/v1/projects_controller.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        else
+          render status: 400, json: { errors: @project.errors }
+        end
+      end
+
+      # PATCH/PUT /projects/1
+      def update
+        if @project.update(project_params)
+          render status: 200, json: { project: @project }
+          # respond_with @project, serializer: ProjectSerializer, status: 200
```
Comment (tomaass) [inline app/controllers/api_frontend/v1/projects_controller.rb]: @ondrejbartas it looks better with `respond_with` but `respond_with` is deprecated in current version of rails & it doesn't work for update, it responds with empty string and status 204 - no content and I don't know why.

Could you take a look at it if we want to keep `respond_with` here?

Comment (ondrejbartas) [reply app/controllers/api_frontend/v1/projects_controller.rb]: Yes it is, they extracted respond_with to gem 'responders'

**Pull request merged**